src/initializer.js

```diff
@@ -0,0 +1,27 @@
+const fs = require('fs');
```

Comment:
If you're targeting Node 10+, I would use the `promise`s version of `fs`, to remove the need for the `fs.existsSync` and callbacks (🤢)

```suggestion
const fs = require('fs/promises');
```

Reply:
I can't seem to get this to work regardless of the node version I am using...